docs/user/Bazel.md

```diff
@@ -4,24 +4,102 @@ The main use-case for Bazel is to make modifications to OpenROAD and run local t
+- `--jobs=$(nproc)` limits parallel builds to number of cores in the system, optional.
+
+Note! You'll see `bazel` in examples and documentation as well as `bazelisk`. The latter is a wafer thin layer on top of `bazel` that reads in the `.bazelversion` file to decide which version of Bazel to use in bazel.
 
 A word on expectations: Bazel is a valuable skill for the future. It is an example of a new generation of build tools(Buck2 is another example) that scales very well, are hermetic and have many features, such as artifacts. Unsurprisingly, this does mean that there is a lot to learn. The OpenROAD documentation makes no attempt at teaching Bazel. Bazel is a very wide topic and it can not be learned in a day or two with intense reading of a well defined document with a start and an end. It is probably best to start as a user running canned commands, such as above, but switch from mechanical repetition of canned command to being curious and following breadcrumbs of interest: read, search, engage with the community and use AI to learn.
 
 ## Running specific tests or tests below a folder
 
-To list all tests:
+To list all tests use bazelisk query with [Bazel query language](https://bazel.build/query/language):
 
     bazelisk query 'kind(test, ...)'
 
+List all tests below a folder containing `asap7`:
+
+    $ bazelisk query   'filter("asap7", kind(".*_test rule", //src/pdn/...))'
+    //src/pdn/test:asap7_M1_M3_followpins-tcl
+    ...
+
 Run specific test:
 
-    baselisk test //src/upf/test:levelshifter-tcl
+    baselisk test --test_output=errors //src/upf/test:levelshifter-tcl
 
 To run or list tests below a folder:
 
     bazelisk test src/gpl/...
 
+## Build configurations
+
+Bazel build configuration is a big topic as it covers cross compilation as well as multiplatform support.
+
+As a start, browse `.bazelrc` and also run:
+
+    $ bazelisk build
+    [wait until it starts building then hist ctrl-c, this warms up the cache with information about available configs]
+    $ bazelisk config
+    Available configurations:
+    5e5e8a80a777eb91e67b2d19d33945262b2897b636da1007246cf68b6b6ec51d k8-fastbuild
+    781deb76199c2d7f1a6c7d54da3ea8dad03e6956c04db58addba49068e4d3797 k8-fastbuild
+    96deab75888eab9e42bcc5778aa824b18bb8ee745dfa1950a18536d75ee05e01 k8-opt-exec-ST-d57f47055a04 (exec)
+    dd6f5325b352d3c63abe45b22d15914f75da7e2f140ace0017440dfc00f49114 k8-opt-exec-ST-d57f47055a04 (exec)
+    f37096aa0a6acd138beca4ff4d66b677012cd1d0d54befaa35983993505dad60 fastbuild-noconfig
+    f6ad5d5ec52c67510c6642c6d8df81fb134760a95c43d1f8e0369ad2c8964a81 k8-opt-exec-ST-6f5a6fb95be7 (exec)
+
+Without offering any deeper insight some comments about what is shown above:
+
+- Each configuration has a textual shorthand used in folder names followed by a full hash
+- `fastbuild` is the default compromise between optimization and fast builds
+- `exec` means host, appears with `ST` and an extra hash at the end
+- `k8` always there, history?
```

Comment:
The K8 was the first x86-64 CPU and is equivalent to it afaik

Reply:
added ref